tests/export.test.ts

```diff
@@ -0,0 +1,38 @@
+import { randomUUID } from "node:crypto";
+import { afterAll, beforeAll, test } from "vitest";
+import { assert, getClient } from "./utils/meilisearch-test-utils.js";
+
+const INDEX_UID = randomUUID();
+const ms = await getClient("Master");
+
+beforeAll(async () => {
+  const task = await ms.createIndex(INDEX_UID).waitTask();
+  assert.isTaskSuccessful(task);
+
+  const task2 = await ms
+    .index(INDEX_UID)
+    .addDocuments([{ id: 0, beep: "boop" }])
+    .waitTask();
+  assert.isTaskSuccessful(task2);
+});
+
+afterAll(async () => {
+  const task = await ms.deleteIndex(INDEX_UID).waitTask();
+  assert.isTaskSuccessful(task);
+});
+
+test(`${ms.export.name} method`, async () => {
+  const task = await ms
+    .export({
+      url: "http://127.0.0.1:7702",
```

Comment:
Would this work in Docker? Shouldn't we use the service's internal name, e.g., `http://export-meilisearch:7700`?

Reply:
Okay, now I get:

```json
{
  "code": "internal",
  "link": "https://docs.meilisearch.com/errors#internal",
  "message": "Failed to export documents to remote server missing_payload (invalid_request): A ndjson payload is missing. <https://docs.meilisearch.com/errors#missing_payload>",
  "type": "internal",
}
```

Not sure what the problem is here.